src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
+   * @returns {*}
+   */
+  function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
```

Comment:
@topherfangio So the `md-autocomplete-wrap-override` seemed like an easy way to do it but then it got fragile when I looked at a situation where the `md-autocomplete` element resided inside and `ng-if`. In this case the `md-autocomplete-wrap-override` can't find the `md-autocomplete` controller to register to.